src/Form/fields/StringField.tsx

```diff
@@ -15,7 +17,29 @@ export class StringField extends React.Component<IWidget> {
   }
 
   public render(): React.ReactNode {
-    const {schema, uiSchema, isRequired, value} = this.props;
+    const {
+      schema,
+      schema: {format},
+      uiSchema,
+      isRequired,
+      value,
+    } = this.props;
+
+    if (format) {
```

Comment:
We can use `if` statement to decide what should be passed to `getWidget` function as an parameter and then return only `Widget` instead of `Widget` or `InputWidget`. (to do that we have to add `InputWidget` to `widgets/index` registry.